arch/x86/core/crt0.S

```diff
@@ -15,6 +15,93 @@
 #include <kernel_arch_data.h>
 #include <arch/cpu.h>
 
+#ifdef CONFIG_JAILHOUSE
```

Comment:
Can you move this below all the GDATA/GTEXT declarations?

Reply:
OK